BTrees/check.py

```diff
@@ -346,13 +348,13 @@ def visit_bucket(self, obj, path, parent, is_mapping,
     def check_sorted(self, obj, path, keys, lo, hi):
         i, n = 0, len(keys)
         for x in keys:
-            if lo is not None and not lo <= x:
+            if lo is not None and not cmp(lo, x) <= 0:
```

Comment:
Maybe worth a comment that `None` is treated as a sentinel in `lo` and `hi` ("without bound") rather than a literal `None`.